logging/src/main/java/org/jboss/as/logging/logmanager/WildFlyLogContextSelectorImpl.java

```diff
@@ -41,7 +42,15 @@
 
     public WildFlyLogContextSelectorImpl() {
         counter = new AtomicInteger(0);
-        contextSelector = new ClassLoaderLogContextSelector(true);
+        // Use the current log context as the default, not LogContext.DEFAULT_LOG_CONTEXT_SELECTOR
+        // This allows embedding use cases to control the log context
+        final LogContext defaultLogContext = LogContext.getLogContext();
+        contextSelector = new ClassLoaderLogContextSelector(new LogContextSelector() {
+            @Override
+            public LogContext getLogContext() {
+                return defaultLogContext;
+            }
+        }, true);
```

Comment:
Is there a reason to wrap this in a new `LogContextSelector` rather than just using `new ClassLoaderLogContextSelector(defaultLogContext, true)`? It doesn't really matter all that much really.